go.mod

```diff
@@ -3,10 +3,14 @@ module github.com/tarantool/go-tarantool
 go 1.11
 
 require (
+	github.com/google/go-cmp v0.5.7 // indirect
 	github.com/google/uuid v1.3.0
 	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
 	github.com/stretchr/testify v1.7.1 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/shopspring/decimal v1.3.1
 	google.golang.org/appengine v1.6.7 // indirect
 	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f // indirect
 	gopkg.in/vmihailenco/msgpack.v2 v2.9.2
+	gotest.tools v2.2.0+incompatible
```

Comment:
May it break compatibility with Go 1.13? Seems that it is not since CI is OK, but how exactly this dependency works?

Reply:
I'm not sure that I understand your question correctly. What do you mean?
CI passed with Go 1.13, what's wrong here?

Reply:
As I get it right, `+incompatible` means the dependency (`gotest.tools` [^2]) has a semver major version of 2 or higher (means that version is stable enough) and is not a Go module yet (it doesn't have `go.mod` in its source code), found explanation in [^1] and [^3]. And `gotest.tools`'s README says that:

> To use gotest.tools with an older version of Go that does not understand Go module paths pin to version v2.3.0.

And `incompatible` flag in `go.mod` looks as a compatibility thing for old versions. The oldest supported Go version is 1.13, that already supports Go modules. I have updated go.tools to v3 and seems the problem is gone:
```diff
--- a/go.mod
+++ b/go.mod
@@ -3,11 +3,15 @@ module github.com/tarantool/go-tarantool
 go 1.11
 
 require (
+       github.com/google/go-cmp v0.5.7 // indirect
        github.com/google/uuid v1.3.0
        github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
+       github.com/pkg/errors v0.9.1 // indirect
+       github.com/shopspring/decimal v1.3.1
        github.com/stretchr/testify v1.7.1 // indirect
        github.com/tarantool/go-openssl v0.0.8-0.20220419150948-be4921aa2f87
        google.golang.org/appengine v1.6.7 // indirect
        gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f // indirect
        gopkg.in/vmihailenco/msgpack.v2 v2.9.2
+       gotest.tools/v3 v3.2.0 // indirect
 )

```

[^1]: https://stackoverflow.com/a/57356777/3665613
[^2]: https://github.com/gotestyourself/gotest.tools
[^3]: https://github.com/golang/go/wiki/Modules#can-a-module-consume-a-v2-package-that-has-not-opted-into-modules-what-does-incompatible-mean

Reply:
> Well, I don't remember why I have asked the question, but it seems that go.mod incompatible does not related to Go version restrictions at all.

To be honest, I'm overlooked this on self-review. Thanks for highlighting!